Library/Homebrew/cask/audit.rb

```diff
@@ -486,7 +486,9 @@ def audit_livecheck_unneeded_long_version
 
     sig { void }
     def audit_signing
-      return if !signing? || download.blank? || (url = cask.url).nil?
+      return if download.blank? || (url = cask.url).nil?
+      return if !cask.tap.official? && !signing?
```

Comment:
@SMillerDev Does this work for you? We skip on non-official taps unless `--signing` is passed. Official tap we don't skip.

Reply:
Sounds good to me!

Reply:
There's an argument long-term that we should make this the default as the unsigned handling will likely bitrot when we're no longer using it in official taps.